core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -273,7 +282,7 @@ <h3>Roles</h3>
                   <select ng-model="newMemberRole" class="form-control protractor-test-role-select" ng-options="r.name for r in ROLES" style="width: 250px;">
                   </select>
                   <span class="help-block">
-                    Note that managers also have the permissions of collaborators, and collaborators also have the permissions of viewers. Please note that assigning roles is irreversible (though you can always assign somebody to a higher role).
+                    Note that managers also have the permissions of collaborators, collaborators also have the permissions of translators, and translators also have permissions of viewers. Please note that assigning roles is irreversible (though you can always assign somebody to a higher role).
```

Comment:
...translators also have *the* permissions of viewers...

Reply:
Done!